### PR TITLE
Charts: enabled newui in skydive-analyzer chart

### PR DESCRIPTION
**Pull request by hunchback**
for now there is a bug as the latest webui does not connect to localhost:8082 successfully

to run:

```
cd charts
make k3d
make skydive-analyzer
make install
make status
```

attempting to logout and login with:

- host: http://localhost:802
- user: admin
- password: password

fails:

image

while access localhost:8082 from within the skydive-ui contains succeeds:

image

### CODE REVIEW

**Comment by hunchback** [comment]
@safchain - please check

other than the bug, I think that from the user perspective it would be best if you would integrate the newui into the skydive binary, as it would be best to just access it via a url such as: http://localhost:8082/newui

**Comment by safchain** [comment]
Yes the plan is to integrate the new UI, but it still need some work. For the issue can you try to see what is the exact error using the web console (network planel) ? I'll push a fix to report the real error at the UI side.

File: .mk/tools.mk [hunk 1]
```diff
@@ -0,0 +1,42 @@
+TOOLSBIN := /usr/local/bin
```
[the remark this answers is not in the document]
Comment (hunchback) [reply .mk/tools.mk:1]: ok

File: charts/Makefile [hunk 2]
```diff
@@ -0,0 +1,11 @@
+SUBDIRS := \
```
[the remark this answers is not in the document]
Comment (hunchback) [reply charts/Makefile:1]: ok

**Comment by hunchback** [comment]
> Yes the plan is to integrate the new UI, but it still need some work. For the issue can you try to see what is the exact error using the web console (network planel) ? I'll push a fix to report the real error at the UI side.

image

**Comment by hunchback** [comment]
> Yes the plan is to integrate the new UI, but it still need some work. For the issue can you try to see what is the exact error using the web console (network planel) ? I'll push a fix to report the real error at the UI side.

I think that the fact the newui is outside the core has had an adverse effect on the newui rate of maturing, IMHO the sooner the newui is merged into the core (mapped to a unique prefix such as /newui) the better.

**Comment by hunchback** [comment]
was only for purposes of reporting a bug

**Pull request closed without merging**